### PR TITLE
change to use evfevent as file extension

### PR DESCRIPTION
**Pull request by william-xiang**
change to use evfevent as file extension

### CODE REVIEW

File: IBMiMake [hunk 1]
```diff
@@ -248,10 +248,10 @@ cleanCDeps = awk '$$2 !~ /^\/QIBM\// && $$2 !~ /$(notdir $<)$$/ && $$2 !~ /$(bas
 # This defines the steps taken after a C compile to massage the auto-generated dependencies into a useable form.
 # See http://make.mad-scientist.net/papers/advanced-auto-dependency-generation/#tldr
 define POSTCCOMPILE =
-$(ICONV) -f $(ICONV_EBCDIC) -t $(ICONV_ASCII) $(DEPDIR)/$*.Td | tr -d '\r' > $(DEPDIR)/$*.T2d
+$(ICONV) -f $(ICONV_EBCDIC) -t $(ICONV_ASCII) $(DEPDIR)/$*.evfevent | tr -d '\r' > $(DEPDIR)/$*.T2d
 $(cleanCDeps) <$(DEPDIR)/$*.T2d | sort -u >$(DEPDIR)/$*.d
 touch -cr $(OBJPATH)/$@ $(DEPDIR)/$*.d
-rm $(DEPDIR)/$*.Td $(DEPDIR)/$*.T2d
+rm $(DEPDIR)/$*.evfevent $(DEPDIR)/$*.T2d
```
Comment (edmundreinhardt) [inline IBMiMake:254]: Do not delete .evfevent files

File: IBMiMake [hunk 2]
```diff
@@ -267,14 +267,14 @@ cleanRPGDeps = awk '$$1 == "FILEID" && $$6 !~ /^QTEMP/ && toupper($$6) !~ /QSYS/
 # convert everything to upper case, format in makefile dependency format, and output all these dependencies
 # to a file that will be included by Make.
 define POSTRPGCOMPILE =
-system "CPYTOSTMF FROMMBR('$(OBJPATH)/EVFEVENT.FILE/$*.MBR') TOSTMF('$(DEPDIR)/$*.Td') STMFCCSID(*STDASCII) ENDLINFMT(*LF) CVTDTA(*AUTO) STMFOPT(*REPLACE)" >/dev/null
-{ for f in $$($(cleanRPGDeps) <$(DEPDIR)/$*.Td | sort -u); do \
+system "CPYTOSTMF FROMMBR('$(OBJPATH)/EVFEVENT.FILE/$*.MBR') TOSTMF('$(DEPDIR)/$*.evfevent') STMFCCSID(*STDASCII) ENDLINFMT(*LF) CVTDTA(*AUTO) STMFOPT(*REPLACE)" >/dev/null
```
Comment (edmundreinhardt) [inline IBMiMake:270]: use EVTDIR

File: IBMiMake [hunk 3]
```diff
@@ -267,14 +267,14 @@ cleanRPGDeps = awk '$$1 == "FILEID" && $$6 !~ /^QTEMP/ && toupper($$6) !~ /QSYS/
...
+{ for f in $$($(cleanRPGDeps) <$(DEPDIR)/$*.evfevent | sort -u); do \
     echo "$${f}"; \
     sed -n -r -e '/^.{6}[^*].{14}[^Ff]/ s/^.{5}[Ff]([^ ]{,10}).*(DISK|WORKSTN|PRINTER).*/  \1\.FILE/I p' -e '/^.{6}[^*]/ s/^.{5}[Dd].*EXTNAME\(([^ \)]*).*/  \1\.FILE/I p' "$${f}" | sort -u; \
   done; \
 } | sed -e 's/^\/.*\///' -e 's/^/$@: /' | tr '[:lower:]' '[:upper:]' >$(DEPDIR)/$*.d
 touch -cr $(OBJPATH)/$@ $(DEPDIR)/$*.d
-#rm $(DEPDIR)/$*.Td
+#rm $(DEPDIR)/$*.evfevent
```
Comment (edmundreinhardt) [inline IBMiMake:277]: Do not delete EVTDIR

File: IBMiMake [hunk 4]
```diff
@@ -267,14 +267,14 @@ cleanRPGDeps = awk '$$1 == "FILEID" && $$6 !~ /^QTEMP/ && toupper($$6) !~ /QSYS/
 # convert everything to upper case, format in makefile dependency format, and output all these dependencies
 # to a file that will be included by Make.
 define POSTRPGCOMPILE =
-system "CPYTOSTMF FROMMBR('$(OBJPATH)/EVFEVENT.FILE/$*.MBR') TOSTMF('$(DEPDIR)/$*.Td') STMFCCSID(*STDASCII) ENDLINFMT(*LF) CVTDTA(*AUTO) STMFOPT(*REPLACE)" >/dev/null
-{ for f in $$($(cleanRPGDeps) <$(DEPDIR)/$*.Td | sort -u); do \
+system "CPYTOSTMF FROMMBR('$(OBJPATH)/EVFEVENT.FILE/$*.MBR') TOSTMF('$(DEPDIR)/$*.evfevent') STMFCCSID(*STDASCII) ENDLINFMT(*LF) CVTDTA(*AUTO) STMFOPT(*REPLACE)" >/dev/null
+{ for f in $$($(cleanRPGDeps) <$(DEPDIR)/$*.evfevent | sort -u); do \
```
Comment (edmundreinhardt) [inline IBMiMake:271]: Use EVTDIR

File: IBMiMake [hunk 5]
```diff
@@ -267,14 +267,14 @@ cleanRPGDeps = awk '$$1 == "FILEID" && $$6 !~ /^QTEMP/ && toupper($$6) !~ /QSYS/
...
     echo "$${f}"; \
     sed -n -r -e '/^.{6}[^*].{14}[^Ff]/ s/^.{5}[Ff]([^ ]{,10}).*(DISK|WORKSTN|PRINTER).*/  \1\.FILE/I p' -e '/^.{6}[^*]/ s/^.{5}[Dd].*EXTNAME\(([^ \)]*).*/  \1\.FILE/I p' "$${f}" | sort -u; \
   done; \
 } | sed -e 's/^\/.*\///' -e 's/^/$@: /' | tr '[:lower:]' '[:upper:]' >$(DEPDIR)/$*.d
```
Comment (edmundreinhardt) [inline IBMiMake:275]: stick with DEPDIR here

File: IBMiMake [hunk 6]
```diff
@@ -536,7 +536,7 @@ programTGTRLS = $(strip \
 	@echo "*** Creating RPG module [$*]"
 	@$(set_STMF_CCSID)
 	$(eval crtcmd := crtrpgmod module($(OBJLIB)/$*) srcstmf('$<') $(CRTRPGMODFLAGS))
-	@system -v "$(TOOLSLIB)/EXECWLIBS LIB($(LIBL)) CMD($(crtcmd))" > $(LOGPATH)/$(notdir $<).log
+	-@system -v "$(TOOLSLIB)/EXECWLIBS LIB($(LIBL)) CMD($(crtcmd))" > $(LOGPATH)/$(notdir $<).log
```
Comment (edmundreinhardt) [inline IBMiMake:539]: don't use '-' prefix instead
```-@system -v "$(TOOLSLIB)/EXECWLIBS LIB($(LIBL)) CMD($(crtcmd))  ||  $(EVFEVENT_DOWNLOAD)" > $(LOGPATH)/$(notdir $<).log
```
where EVFEVENT_DOWNWLOAD is
 system "CPYTOSTMF FROMMBR('$(OBJPATH)/EVFEVENT.FILE/$*.MBR') TOSTMF('$(DEPDIR)/$*.evfevent') STMFCCSID(*STDASCII) ENDLINFMT(*LF) CVTDTA(*AUTO) STMFOPT(*REPLACE)" >/dev/null

File: IBMiMake [hunk 7]
```diff
@@ -545,15 +545,15 @@ programTGTRLS = $(strip \
 	@$(set_STMF_CCSID)
 	@qsh_out -c "touch -C 1252 $<-1252 && cat $< >$<-1252"
 	$(eval crtcmd := crtsqlci obj($(OBJLIB)/$*) srcstmf('$<-1252') $(CRTSQLCIFLAGS))
-	@system -v "$(TOOLSLIB)/EXECWLIBS LIB($(LIBL)) CMD($(crtcmd))" >$(LOGPATH)/$(notdir $<).log 2>&1 || (ret=$$?; rm $(DEPDIR)/$*.Td 2>/dev/null; rm "$<-1252" 2>/dev/null; exit $$ret)
-	@$(POSTCCOMPILE)
+	@system -v "$(TOOLSLIB)/EXECWLIBS LIB($(LIBL)) CMD($(crtcmd))" >$(LOGPATH)/$(notdir $<).log 2>&1 || (ret=$$?; rm $(DEPDIR)/$*.evfevent 2>/dev/null; rm "$<-1252" 2>/dev/null; exit $$ret)
+	-@$(POSTCCOMPILE)
```
Comment (edmundreinhardt) [inline IBMiMake:549]: this - will not do what you want

File: IBMiMake [hunk 8]
```diff
@@ -545,15 +545,15 @@ programTGTRLS = $(strip \
...
 	@rm "$<-1252"
 
 %.MODULE: %.SQLRPGLE $(DEPDIR)/%.d
 	@echo "*** Creating SQLRPGLE module [$*]"
 	@$(set_STMF_CCSID)
 	$(eval crtcmd := crtsqlrpgi obj($(OBJLIB)/$*) srcstmf('$<') $(CRTSQLRPGIFLAGS))
-	@system -v "$(TOOLSLIB)/EXECWLIBS LIB($(LIBL)) CMD($(crtcmd))" >$(LOGPATH)/$(notdir $<).log 2>&1
+	-@system -v "$(TOOLSLIB)/EXECWLIBS LIB($(LIBL)) CMD($(crtcmd))" >$(LOGPATH)/$(notdir $<).log 2>&1
```
Comment (edmundreinhardt) [inline IBMiMake:556]: again add || $(EVFEVENT_DOWNLOAD) and don't us -

File: IBMiMake [hunk 9]
```diff
@@ -248,10 +248,10 @@ cleanCDeps = awk '$$2 !~ /^\/QIBM\// && $$2 !~ /$(notdir $<)$$/ && $$2 !~ /$(bas
 # This defines the steps taken after a C compile to massage the auto-generated dependencies into a useable form.
 # See http://make.mad-scientist.net/papers/advanced-auto-dependency-generation/#tldr
 define POSTCCOMPILE =
-$(ICONV) -f $(ICONV_EBCDIC) -t $(ICONV_ASCII) $(DEPDIR)/$*.Td | tr -d '\r' > $(DEPDIR)/$*.T2d
+$(ICONV) -f $(ICONV_EBCDIC) -t $(ICONV_ASCII) $(DEPDIR)/$*.evfevent | tr -d '\r' > $(DEPDIR)/$*.T2d
```
[the remark this answers is not in the document]
Comment (edmundreinhardt) [reply IBMiMake]: I am sorry, I steered you wrong on this line

**Review by edmundreinhardt — APPROVED** [review]

File: IBMiMake [hunk 10]
```diff
@@ -248,10 +250,10 @@ cleanCDeps = awk '$$2 !~ /^\/QIBM\// && $$2 !~ /$(notdir $<)$$/ && $$2 !~ /$(bas
 # This defines the steps taken after a C compile to massage the auto-generated dependencies into a useable form.
 # See http://make.mad-scientist.net/papers/advanced-auto-dependency-generation/#tldr
 define POSTCCOMPILE =
-$(ICONV) -f $(ICONV_EBCDIC) -t $(ICONV_ASCII) $(DEPDIR)/$*.evfevent | tr -d '\r' > $(DEPDIR)/$*.T2d
+$(ICONV) -f $(ICONV_EBCDIC) -t $(ICONV_ASCII) $(DEPDIR)/$*.Td | tr -d '\r' > $(DEPDIR)/$*.T2d
 $(cleanCDeps) <$(DEPDIR)/$*.T2d | sort -u >$(DEPDIR)/$*.d
 touch -cr $(OBJPATH)/$@ $(DEPDIR)/$*.d
-rm $(DEPDIR)/$*.evfevent $(DEPDIR)/$*.T2d
+rm $(DEPDIR)/$*.T2d
```
[the remark this answers is not in the document]
Comment (william-xiang) [reply IBMiMake:256]: Thank you. I will add this.

**Pull request merged**